SimPEG/Fields.py

```diff
@@ -235,7 +234,9 @@ def _correctShape(self, name, ind, deflate=False):
     def _setField(self, field, val, name, ind):
         srcInd, timeInd = ind
         shape = self._correctShape(name, ind)
-        if Utils.isScalar(val):
+        if isinstance(val, np.ndarray) and val.size == 1:
```

Comment:
This was the only problem I found that was different in upgrading our `isScalar`